recipe/meta.yaml

```diff
@@ -13,16 +13,18 @@ source:
 
 build:
   number: 0
-  script: python setup.py build install --single-version-externally-managed --record record.txt
+  script: "{{ PYTHON }} -m pip install . -vv --global-option build --global-option --force-cython "
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
+    - cython
+    - pip
     - python
     - setuptools
-    - numpy
+    - numpy >=1.12
```

Comment:
This is interesting `1.14` was the prior pin, so this is back pinning?

Reply:
From what I can tell numpy was pinned at 1.11, which resulted in a build error because setuptools was trying to download numpy.

Reply:
So merge this one in?